src/stk/databases/value.py

```diff
@@ -22,6 +22,29 @@ class ValueDatabase:
     :mod:`molecule_value_database <.databases.value>`,
     can serve as good examples.
 
+    *Iterating Through Entries in Database*
+
+    The :meth:`.get_entries` method of a molecule database instance can
+    be used to iterate through entries, the keys of which can be used
+    to access values.
+
+    .. code-block:: python
+
+        client = pymongo.MongoClient()
+        molecule_db = stk.MoleculeMongoDb(client)
+
+        value_db = stk.ValueMongoDb(
+            mongo_client=client,
+            collection='atom_counts',
+        )
+
+        for key, molecule in molecule_db.get_all():
```

Comment:
`get_all()` does not yield the key as currently implemented. Plus there are multiple possible keys.

Reply:
Updated.